docs/docs/guides/accounts-permissions.mdx

```diff
@@ -0,0 +1,138 @@
+2. Click on **Admin** in the left side menu.
+3. Navigate to the **Role Management** section.
+4. In the **Accounts** tab, click on **Create Account**.
+5. Fill in the account's details (name, email, and password).
+6. Optionally, assign the account to a group.
+7. Click **Create** to create the account.
+
+  </TabItem>
+
+  <TabItem value="graphql" label="Via the GraphQL Interface">
+
+In the GraphQL sandbox, execute the following mutation to create a new account, replacing the appropriate values as needed:
+
+    ```graphql
+    mutation AddAccount {
+      CoreAccountCreate(
+        data: {
+          name: {value: "<ACCOUNT-NAME>"},
+          password: {value: "<ACCOUNT-PASSWORD>"}
+          # Optional - Assign the account to an existing group
+          member_of_groups: [{hfid: "Infrahub Users"}]
+        }
+      ) {
+        ok
+        object {
+          hfid
+        }
+      }
+    }
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Creating a new account group
+
+<Tabs>
+  <TabItem value="web" label="Via the Web Interface" default>
+
+1. Login to Infrahub's web interface as an administrator.
+2. Click on **Admin** in the left side menu.
+3. Navigate to the **Role Management** section.
+4. In the **Groups** tab, click on **Create Account Group**.
+5. Enter a name for the group.
+6. Optionally, assign roles to the group.
+7. Click **Create** to create the group.
+
+  </TabItem>
+
+  <TabItem value="graphql" label="Via the GraphQL Interface">
+
+In the GraphQL sandbox, execute the following mutation to create a new group:
+
+    ```graphql
+    mutation AddGroup {
+      CoreAccountGroupCreate(
+        data: {
+          name: {value: "<GROUP-NAME>"},
+          # Optional - Assign existing roles
+          roles: [{hfid: "General Access"}]
+        }
+      ) {
+        ok
+        object {
+          hfid
+        }
+      }
+    }
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Creating and assigning roles
+
+<Tabs>
+  <TabItem value="web" label="Via the Web Interface" default>
+
+1. Login to Infrahub's web interface as an administrator.
+2. Click on **Admin** in the left side menu.
+3. Navigate to the **Role Management** section.
+4. In the **Roles** tab, click on **Create Account Role**.
+5. Provide a name for the role.
+6. Select the permissions you wish to assign to the role.
+7. Optionally, assign the role to an existing group.
+8. Click **Create** to create the role.
+
+  </TabItem>
+
+  <TabItem value="graphql" label="Via the GraphQL Interface">
+
+In the GraphQL sandbox, execute the following mutation to create a new role:
+
+    ```graphql
+    mutation AddRole {
+      CoreAccountRoleCreate(
+        data: {
+          name: {value: "<ROLE-NAME>"},
+          # Optional - Assign the role to an existing group
+          groups: [{hfid: "Infrahub Users"}]
+        }
+      ) {
+        ok
+        object {
+          hfid
+        }
+      }
+    }
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Managing permissions
```

Comment:
How can I create permission object then?

Reply:
Added. Even if the mutation are not 100% the good ones. Considered the incoming PR from @gmazoyer (uniqueness_constraint, removal of name, etc) as already here to not redo it